consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/validator/ForkingValidatorProvider.java

```diff
@@ -79,32 +74,6 @@ public ForkingValidatorProvider(
     return resolveValidatorProvider(header.getNumber() + 1).getVoteProviderAtHead();
   }
 
-  private Collection<Address> getValidators(
-      final ValidatorProvider validatorProvider,
-      final long blockNumber,
-      final Function<ValidatorProvider, Collection<Address>> getValidators) {
-
-    final BftForkSpec<QbftConfigOptions> forkSpec = forksSchedule.getFork(blockNumber);
-
-    // when moving to a block validator the first block needs to be initialised or created with
-    // the previous block state otherwise we would have no validators
-    // unless the validators are being explicitly overridden
-    if (forkSpec.getConfigOptions().isValidatorBlockHeaderMode()
-        && !blockValidatorProvider.hasValidatorOverridesForBlockNumber(blockNumber)) {
```

Comment:
Can the BlockValidatorProvider.hasValidatorOverridesForBlockNumber method also be removed. It shouldn't be necessary now.